Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,35 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// You can bypass SSL peer verification altogether (not advised)
+    public static var verifyPeerSSLCertificate = true
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the path
+    /// provided by setting URLSession.sslCertificateAuthorityFile to a URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL {
+        set(value) {
+            verifyPeerSSLCertificate = true
```

Comment:
You’ll need to correct the typos in that comment and use consistent terminology (i.e., “certificate authority file”).

However, this design is not the right way to go, IMO. You’ve got to commit to your convictions. If you believe that there is some configuration that is insecure, then it should be prevented and not merely made unlikely. That is, setting a non-nil CA file should be a fatal error when the other setting is false, and changing the other setting to false should be a fatal error too. On the other hand, you can say that the configuration is unwise but legitimate; in that case, there should not be some arbitrary sequence of setting properties that prevents it but others that don’t.